### PR TITLE
CASMINST-6222: System recovery-related linting and link fixing

### PR DESCRIPTION
**Pull request by mharding-hpe**
# Description
Fixing up some links that broke with a recent PR, and doing other minor linting.

[1.4 backport PR](https://github.com/Cray-HPE/docs-csm/pull/3647)

No earlier backports needed.

# Checklist

- [X] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [X] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [X] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Comment by kimjensen-hpe** [comment]
Also note that in the backport to release/1.4, some of the procedures related to postgres backups are using the old home-grown backups - where as main is using the logical backups based on the upgraded postgres operator. So expect merge conflicts if you are backporting main to 1.4 as the 1.4 content needs to remain as is relating to postgres

File: operations/security_and_authentication/Keycloak_Service_Recovery.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ The following covers redeploying the Keycloak service and restoring the data.
 - All activities required for site maintenance are complete.
 - A backup or export of the data already exists.
 - The latest CSM documentation has been installed on the master nodes. See [Check for Latest Documentation](../../update_product_stream/index.md#check-for-latest-documentation).
+- The Cray CLI has been configured on the node where the procedure is being performed. See [Configure the Cray CLI](../configure_cray_cli.md).
```
[the remark this answers is not in the document]
Comment (mharding-hpe) [reply operations/security_and_authentication/Keycloak_Service_Recovery.md:11]: Good catch. Ok, I'll modify this page to match that one in that respect.

**Comment by mharding-hpe** [comment]
> Also note that in the backport to release/1.4, some of the procedures related to postgres backups are using the old home-grown backups - where as main is using the logical backups based on the upgraded postgres operator. So expect merge conflicts if you are backporting main to 1.4 as the 1.4 content needs to remain as related to postgres backups

I think I handled this in the backport, but I've added you as a reviewer so you can confirm.

File: operations/kubernetes/Restore_Postgres.md [hunk 2]
```diff
@@ -26,8 +26,8 @@ In the event that the Spire Postgres cluster must be rebuilt and the data restor
 
     * If a manual dump of the database was taken, then check that the dump file exists in a location off the Postgres cluster. It will be needed in the steps below.
     * If the database is being automatically backed up, then the most recent version of the dump and the secrets should exist in the `postgres-backup` S3 bucket.
-    These will be needed in the steps below. List the files in the `postgres-backup` S3 bucket and if the files exist, download the dump and secrets out of the S3 bucket.
-    The `cray artifacts` CLI can be used list and download the files. Note that the `.psql` file contains the database dump and the .manifest file contains the secrets.
+      These will be needed in the steps below. List the files in the `postgres-backup` S3 bucket and if the files exist, download the dump and secrets out of the S3 bucket.
+      The `cray artifacts` CLI can be used list and download the files. Note that the `.psql` file contains the database dump and the `.manifest` file contains the secrets.
```
Comment (kimjensen-hpe) [inline operations/kubernetes/Restore_Postgres.md:30]: "CLI can be used to list ..."
The "to" is missing here and in a few other places below.

**Review by kimjensen-hpe — APPROVED** [review]

File: operations/kubernetes/Restore_Postgres.md (hunk 2 above)
Comment (mharding-hpe) [reply operations/kubernetes/Restore_Postgres.md]: Good catch. Fixed.

**Review by nrockershousen — APPROVED** [review]

**Pull request merged**